File: exercise_4.py
import math
a = input("Enter the a value ")
a=int(a)
b = input("Enter the b value ")
b=int(b)
c = input("Enter the c value ")
c=int(c)
if (b**2-(4*a*c)) < 0:
    print ("Dear user unfortunately there are no real-valued solutions that exist for this equation")
else:
    d = b**2-4*a*c
    x1 = ((((-1)*b)+math.sqrt(d)))/(2*a)
    x2 = ((((-1)*b)-math.sqrt(d)))/(2*a)
    print ("The first solution is:", x1)
    print ("The second solution is:", x2)

File: Coding Bootcamp/11_10_2016 Java/Python to Java exercises from Python 27_09_2016/exercise_4.java
import java.util.Scanner;
public class Exercise5 {

    public static void main (String [] args) {
        Scanner userinput = new Scanner (System.in);
        System.out.println("Enter the a value of the equation: ");
        int a = userinput.nextInt();
        System.out.println("Enter the b value of the equation: ");
        int b = userinput.nextInt();
        System.out.println("Enter the c value of the equation: ");
        int c = userinput.nextInt();
        if (Math.pow(b,2) - (4*a*c) < 0) {
          System.out.println("Dear user unfortunately there are no real-valued solutions that exist for this equation");
      } else {
          double d = Math.pow(b,2) - 4*a*c;
          double x1 = ((((-1)*b)+Math.sqrt(d)))/(2*a);
          double x2 = ((((-1)*b)-Math.sqrt(d)))/(2*a);
          System.out.println("The first solution is:" + x1);
          System.out.println("The first solution is:" + x2);
      }
    }
}
